tools/cardano-sl-tools.cabal

```diff
@@ -17,6 +17,12 @@ Flag with-web
 
   description: Build with web part.
 
+Flag with-post-mortem-tool
+  default:     False
+  manual:      True
+
+  description: Build with web part.
```

Comment:
@gromakovsky See [here](https://github.com/input-output-hk/cardano-sl/pull/1116#issuecomment-319408843) for the context. I don't care which flag we use in the end, provided we can easily disable building the `post-mortem-tool` tool on CI but still build it locally for our tests. I'm not familiar with `for-installer` and what it does, but if it will install more stuff what I really need (aka just the post-mortem tool) I would be inclined to leave the flag be. The description is wrong, yes, and should be fixed.